spring-cloud-sleuth-stream/src/main/java/org/springframework/cloud/sleuth/stream/ServerPropertiesHostLocator.java

```diff
@@ -38,12 +42,14 @@
 
 	private final ServerProperties serverProperties; // Nullable
 	private final String appName;
+	private final InetUtils inetUtils;
 	private Integer port; // Lazy assigned
 
-	public ServerPropertiesHostLocator(ServerProperties serverProperties,
-			String appName) {
+	public ServerPropertiesHostLocator(ServerProperties serverProperties, String appName,
```

Comment:
I'd delegate the old constructor to this one, so it isn't an api break. Ex. by newing up InetUtils when the old constructor is used.